src/aap_eda/core/migrations/0009_add_git_hash_to_activations.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.20 on 2023-08-29 21:17
+
+from django.db import migrations, models
+
+
+def add_git_hash_to_activation(apps, schema_editor):
+    Activation = apps.get_model("core", "Activation")  # noqa: N806
+
+    for activation in Activation.objects.all():
+        git_hash = "project deleted"
+        if activation.rulebook and activation.rulebook.project:
+            git_hash = activation.rulebook.project.git_hash
+        activation.git_hash = git_hash
+        activation.save(update_fields=["git_hash"])
```

Comment:
@Dostonbek1 do we need this for activation_instance.git_hash too?

Reply:
there is no way no know what was the hash for the instance

Reply:
This could be inaccurate. IMHO Leaving it blank might be better than an inaccurate value

Reply:
@bzwei agreed.